packages/financial-templates-lib/src/helpers/GasEstimator.js

```diff
@@ -2,42 +2,51 @@
 // to inform the Liquidator and dispute bot of a reasonable gas price to use.
 
 const fetch = require("node-fetch");
-// Etherchain expected response structure:
-// {
-//   "safeLow": "25.0",
-//   "standard": "30.0",
-//   "fast": "35.0",
-//   "fastest": "39.6"
-// }
-const url = "https://www.etherchain.org/api/gasPriceOracle";
-// Etherscan API limits 1 request every 3 seconds without passing in an API key. Expected response structure:
-// {
-//   "status": "1",
-//   "message": "OK-Missing/Invalid API Key, rate limit of 1/3sec applied",
-//   "result": {
-//       "LastBlock": "10330323",
-//       "SafeGasPrice": "30",
-//       "ProposeGasPrice": "41"
-//   }
-// }
-const backupUrl = "https://api.etherscan.io/api?module=gastracker&action=gasoracle";
 
+const GAS_ESTIMATOR_MAPPING_BY_NETWORK = {
+  // Expected shape:
+  // <netId>: {
+  //     url: <primary-gas-station-url>,
+  //     backupUrl: <optional-backup-gas-station-url>,
+  //     defaultFastPricesGwei: <default-gas-price-for-network>
+  // }
+  1: {
+    url: "https://www.etherchain.org/api/gasPriceOracle",
+    backupUrl: "https://api.etherscan.io/api?module=gastracker&action=gasoracle",
+    defaultFastPriceGwei: 50,
+  },
+  137: {
+    url: "https://gasstation-mainnet.matic.network",
+    defaultFastPriceGwei: 10,
+  },
+  80001: {
+    url: "https://gasstation-mumbai.matic.today",
+    defaultFastPriceGwei: 20,
+  },
+};
+
+const DEFAULT_NETWORK_ID = "1"; // Ethereum Mainnet.
```

Comment:
do you want the default to be a string? should it not be a number?

Reply:
Good point, changed everything to int's